test/functional/legacy/packadd_spec.lua

```diff
@@ -52,8 +50,10 @@ describe('packadd', function()
         call assert_equal(77, g:plugin_also_works)
         call assert_true(17, g:ftdetect_works)
         call assert_true(len(&rtp) > len(rtp))
-        call assert_true(&rtp =~ (s:plugdir . '\($\|,\)'))
-        call assert_true(&rtp =~ (s:plugdir . '/after$'))
+        if !has('win32')
+          call assert_true(&rtp =~ (s:plugdir . '\($\|,\)'))
```

Comment:
Were these failing for a reason other than differing path separators?

Reply:
Only path separators. `escape` or `substitute` should be enough unless a directory name contains `$#~`

Reply:
If there's an existing Vim patch for this, I can work on it on a separate PR. I don't want to exert more effort for legacy tests.